composer.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "drupal/500px",
+  "name": "drupal/d500px",
```

Comment:
I am not sure this is correct :) 
I was also confused about this, I think this "name" comes from the URL and i had setup the url as 500px. However, since module names cannt begin with numbers, I had to call the module D500px. 

> Projects must use the package name:
> drupal/PROJECT
> where PROJECT is the part from the URL.

https://www.drupal.org/node/2471927